packages/preset-create-react-app/README.md

```diff
@@ -26,15 +26,35 @@ In most cases, this preset will find your `react-scripts` package, even if it's
 
 In the event that it doesn't, you can set the package's name with `scriptsPackageName`.
 
-You can also enable and configure `react-docgen-typescript-loader`.
+You can also enable and configure [`react-docgen-typescript-loader`](https://github.com/strothj/react-docgen-typescript-loader) with `tsDocgenLoaderOptions`.
+
+If set to `{}`, it will be enabled with default Create React App settings.
+
+```js
+module.exports = [
+  {
+    name: '@storybook/preset-create-react-app',
+    options: {
+      scriptsPackageName: '@my/react-scripts',
+      tsDocgenLoaderOptions: true,
```

Comment:
`true`? or `{}`? i'm confused

Reply:
Sorry, that was me playing around and I forgot to revert. Fixed.